src/gt4sd/cli/load_arguments_from_dataclass.py

```diff
@@ -103,7 +109,7 @@ def extract_fields_from_class(
                 raise ValueError(f"{arg_fields[field_name]['type']} not supported")
 
         else:
-            raise ValueError(
+            logger.error(
```

Comment:
why not raising as done above?

Reply:
If we raise, the tests in `test_training_pipelines.py` fail with errors like:
```sh
 ValueError:  argument evaluation_strategy: <enum 'IntervalStrategy'> not supported
```
Our parsers are quite restrictive on supported argument types for the dataclasses which conflicts with the HF Training arguments that support arbitrary types: https://github.com/huggingface/transformers/blob/6aae59d0b54f04c13a79f80b708622db8e8a17e4/src/transformers/training_args.py#L496


IMHO this is solved incorrectly in HF because the type annotation should be `Union[str, IntervalStrategy]` rather than `IntervalStrategy`. But since the RT trainer embraces all HF training arguments this type annotation is propagated down and raises in the tests. I think the best way would be to solve this upstream in HF. I can prepare a PR there, WDYT?

Reply:
I think this would be an ideal solution, may I suggest leaving a comment before the `logger.error` saying something like:
```
# NOTE: not raising since the HF training args might introduce typing inconsistencies
```

Reply:
PR opened here: https://github.com/huggingface/transformers/pull/17934

But I'm not sure this can be merged given the HF and argparse requirements, but I will try.

Reply:
Added the comment, as you suggested see the last commit